### ISSUE TITLE
Redesign deck UI

### ISSUE THREAD

**Issue by raatmarien** [raatmarien/playcardswith.me#127]
I want to add a deal feature, but I noticed that the dropdown menu on decks is kind of hard to see. It doesn't really stand out on the decks. And I think that certain deck features, like deal, shuffle and recall, should be really visible to users. So we should think of a way to make it more obvious what the options are I think.

**Comment by raatmarien** [comment]
I made a bit of a redesign proposal, but I'm not sure about it. I moved the decks and the buttons into a big card, giving it a lot more space. But I think it might be a bit to in your face and confusing. What do you all think? 

![2020-04-04-113248_720x574_scrot](https://user-images.githubusercontent.com/7541511/78423640-08879100-7668-11ea-8c54-c59e1eaf25b4.png)

**Comment by raatmarien** [comment]
![2020-04-04-113338_682x571_scrot](https://user-images.githubusercontent.com/7541511/78423658-28b75000-7668-11ea-947f-040f8f79e81d.png)

I gave all the buttons tooltips to explain it to users less familiar with our terminology, which I think is really helpful. But maybe we could do the same with the current drop down

**Comment by raatmarien** [comment]
What do you think @jchmrt @flori9 @sambeard ?

**Comment by raatmarien** [comment]
You can tinker with the code in https://github.com/raatmarien/playcardswith.me/pull/128

**Comment by sambeard** [comment]
First of all I think the remove button should just be a cross icon or smth in the top right corner. Maybe than you can stack the remaining options.

**Comment by flori9** [comment]
I agree that these features should be more visible. But I think this design just takes up a bit too much space. Also it makes it less clear how to even get cards from the deck as the deck itself is in a bit of a weird position.

Maybe we could do something more like this?
![image](https://user-images.githubusercontent.com/13628266/78423777-070a9880-7669-11ea-9901-f80f9236e45e.png)
(this is just a quick paint drawing to give the idea)
So add some more icons on the deck itself.

Alternatively, we could use a UI like yours but use icons instead of text, that would already make it a lot more light.

**Comment by sambeard** [comment]
what if you place the "more icon" next to the deck instead of on top of it? use a  ```v```
I believe we shouldn't clutter the UI too much and also the basic deck functionality works fine without it.

**Comment by flori9** [comment]
By the way, I think the deal feature is far more common and important than the others, especially if we add a "recall all cards first" option to it. Maybe we could put a prominent "Deal" button on the deck and keep the others in a (possibly more visible) overflow menu?
